core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -762,51 +746,79 @@ public static Builder builder(MetaData metaData) {
                                     DiscoverySettings.PUBLISH_TIMEOUT,
                                     InternalClusterService.SETTING_CLUSTER_SERVICE_SLOW_TASK_LOGGING_THRESHOLD);
 
+
     /** As of 2.0 we require units for time and byte-sized settings.  This methods adds default units to any cluster settings that don't
```

Comment:
Maybe fix the javadocs to remove `cluster` since it's now more generic to any provided byte and time settings?